hub/views/tools.py

```diff
@@ -0,0 +1,14 @@
+from django.http import HttpRequest, HttpResponse
+from django.shortcuts import render
+from django.views import View
+
+
+class PostcodeView(View):
```

Comment:
I'd maybe just stick this in the core views file for now. we can split it out if we're going to be adding more tools.